### ISSUE TITLE
Fix the Unauthorized for url Error

### ISSUE THREAD

**Issue by efencee** [projectstake/git-notion#11]
The GitHub Action to run git-notion when PRs are merged is still failing despite the addition of the Secrets.

**Comment by ikezue** [comment]
I switched the secret to use the v2 token and ran the job again. Seems it now works.

**Comment by efencee** [comment]
All right, I'll test it on my end as well.

**Comment by efencee** [comment]
I deleted the previous forked version I had, and forked this repository again but the sync action is still failing on my end. @Steph-crown could you try this on your end?

**Comment by ikezue** [comment]
Is this still an issue?

**Comment by Steph-crown** [comment]
I don't think so since syncing works well now. I'll go ahead and close the issue.
